rxfingerprint/src/main/java/com/mtramin/rxfingerprint/RxFingerprint.java

```diff
@@ -413,7 +415,7 @@ public boolean hasEnrolledFingerprints() {
     }
 
     public void launchFingerprintEnrollment() {
-		context.startActivity(new Intent("ACTION_FINGERPRINT_ENROLL"));
+		context.startActivity(new Intent(ACTION_FINGERPRINT_ENROLL));
```

Comment:
As the constant is only available in the Android P dev preview, should we use the string constant directly instead? `"android.settings.FINGERPRINT_ENROLL"`